### PR TITLE
Doc 698 update database schema

### PR DESCRIPTION
**Pull request by JungaBunga**
Updated DB schema to include tables for new product of SFI-23 quarterly payments

### CODE REVIEW

**Review by samplackett — APPROVED** [review]
excellent-happy

File: changelog/db.changelog-1.26.xml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            <column name="paymentReference" type="VARCHAR(30)">
+                <constraints primaryKey="true" nullable="false" primaryKeyName="daxId_pkey"/>
+            </column>
+            <column name="calculationId" type="INTEGER">
+                <constraints nullable="false"/>
+            </column>
+            <column name="paymentPeriod" type="VARCHAR(200)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="totalQuarterlyPayment" type="NUMERIC">
```
Comment (GodsonLeigh) [inline changelog/db.changelog-1.26.xml:17]: should be paymentAmount

File: changelog/db.changelog-1.26.xml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                <constraints nullable="false"/>
+            </column>
+            <column name="transDate" type="TIMESTAMP">
```
Comment (GodsonLeigh) [inline changelog/db.changelog-1.26.xml:20]: should be transactionDate

File: changelog/db.changelog-1.26.xml [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    </changeSet>
+    <changeSet id="4" author="Amirs Ismuhametovs">
+        <createTable tableName="actions">
+            <column name="actionId" type="INTEGER">
+                <constraints primaryKey="true" primaryKeyName="actionsId_pkey"/>
+            </column>
+            <column name="calculationId" type="INTEGER">
+              <constraints nullable="false"/>
+            </column>
+            <column name="sbi" type="INTEGER">
```
Comment (GodsonLeigh) [inline changelog/db.changelog-1.26.xml:83]: sbi should not be in here

**Review by GodsonLeigh — CHANGES_REQUESTED** [review]

**Pull request merged**